src/LiteCore/src/LiteCore.Shared/Interop/C4Database.cs

```diff
@@ -108,4 +108,100 @@ public void Dispose()
             storageEngine = null;
         }
     }
+
+    [ExcludeFromCodeCoverage]
+    internal sealed unsafe class DatabaseConfig2 : IDisposable
```

Comment:
I'm unclear what the purpose of this class is at this point.  It doesn't seem to be doing anything substantial and I think it's going to lead to a lot of confusing corner cases later rather than keeping the abstraction thin like C4DatabaseConfig.